### DISCUSSION TITLE
How to disable a keybinding?

### DISCUSSION THREAD

**Discussion by claudiofreitas** [jesseduffield/lazygit#2150] [Q&A] [answered]
Is there any way to disable a keybinding?
I wanted to disable most of them and enable them as I discover my own needs,
also to avoid pressing some key by accident and have a blank slate not having to worry about key clashes.

I tried the snippet below to see if empty string would do the trick, but the app crashes.

```yaml
keybinding:
  universal:
    quit: ''
```

**Comment by mark2185** [comment, marked as answer]
I don't think there is a `<nop>` of some sort, but I guess you could remap it to something rarely used.
E.g. `<a-enter>` (`alt+enter`), at least until a `<nop>` keybinding is implemented.

_Or until someone else comes and corrects me about there not being such a feature yet_

**Reply by claudiofreitas** [reply]
Hi @mark2185, thank you very much for the suggestion.
Would you know where would be the best place to request this feature to be added?
Just leaving this thread here would be enough?
I do not know go, otherwise I might attempt to contribute to the project.

**Reply by mark2185** [reply]
Turned it into an issue, someone will pick it up sooner or later.

If you learn `go` in the meantime, feel free to try and implement it :)

**Reply by claudiofreitas** [reply]
Thank you once again!

**Reply by mark2185** [reply]
It was solved in #2222, but that's not released yet (as of time of writing).
